src/indra_cogex/client/queries.py

```diff
@@ -346,6 +351,31 @@ def get_pathways_for_gene(client: Neo4jClient, gene: Tuple[str, str]) -> Iterabl
     )
 
 
+def get_shared_pathways_for_genes(
+    client: Neo4jClient, genes: Iterable[Tuple[str, str]]
```

Comment:
can you please make the client keyword only for all new queries

Reply:
see the autoclient decorator branch #60 which will need this

Reply:
Done